dokku-update

```diff
@@ -67,74 +67,91 @@ main() {
     DOKKU_DISTRO=$(. /etc/os-release && echo "$ID")
   fi
 
-  if [[ "$COMMAND" == "version" ]] || [[ "$COMMAND" == "-v" ]]; then
-    VERSION=UNRELEASED
-    if [[ -f "/var/lib/dokku-update/VERSION" ]]; then
-      VERSION="$(cat /var/lib/dokku-update/VERSION)"
-    fi
-    echo "dokku-update ${VERSION}"
-    exit 0
+  if [[ "$PARAMETER" == "-s" ]]; then
+    dokku-log-info "Running system updates"
+    SYSTEM_UPDATES=true
+  else
+    dokku-log-info "Updating Dokku"
+    SYSTEM_UPDATES=false
   fi
 
-  if [[ "$COMMAND" == "run" ]]; then
-    if [[ "$PARAMETER" == "-s" ]]; then
-      dokku-log-info "Running system updates"
-      SYSTEM_UPDATES=true
-    else
-      dokku-log-info "Updating Dokku"
-      SYSTEM_UPDATES=false
-    fi
+  case "$DOKKU_DISTRO" in
+    arch)
+      if [ "$SYSTEM_UPDATES" == true ]; then
+        yay -Syyua
+      else
+        yay -Syu docker-image-labeler dokku nginx plugn sshcommand
+      fi
+      ;;
+    debian | ubuntu)
+      apt-get update -qq >/dev/null
+
+      if [ "$SYSTEM_UPDATES" == true ]; then
+        apt-get -qq -y dist-upgrade
+      else
+        apt-get install --only-upgrade docker-image-labeler dokku dokku-event-listener dokku-update gliderlabs-sigil herokuish nginx plugn sshcommand
+      fi
+      ;;
+    centos | opensuse | rhel)
+      dokku-log-warn "Updating this operating system is not supported"
```

Comment:
We should add centos support here or just decide to kill it entirely?

Reply:
I wonder… CentOS is not mentioned in the landing page’s Quick-start instructions (only Debian and Arch) and I think I heard something about CentOS becoming a rolling-updates type distribution, so I wonder if a lot of people will migrate away in disgust, because that sounds like something people would not like about CentOS… What do you think? I don’t mind either way really, just need a list of packages in case we keep supporting it.

Reply:
Lets support it until we decide to EOL CentOS. It doesn't cost me anything to support atm.

Here is the fpm command which includes centos deps: https://github.com/dokku/dokku/blob/master/rpm.mk#L28

Reply:
Okay! Will add!

Reply:
Added but not really tested 😬 Just went through CentOS/RHEL/OpenSUSE package manager documentation and made an educated guess…